feconf.py

```diff
@@ -1055,3 +1055,149 @@ def get_empty_ratings():
 
 # Authentication method using GAE ID (google sign in)
 AUTH_METHOD_GAE = 'gae'
+
+# The reserved prefix for keys that are automatically inserted into a
+# commit_cmd dict by this model.
+AUTOGENERATED_PREFIX = 'AUTO'
+
+# The command string for a revert commit.
+CMD_REVERT_COMMIT = '%s_revert_version_number' % AUTOGENERATED_PREFIX
+
+# The command string for a delete commit.
+CMD_DELETE_COMMIT = '%s_mark_deleted' % AUTOGENERATED_PREFIX
+
+# IMPORTANT: Ensure that all changes to how these cmds are interpreted preserve
+# backward-compatibility with previous exploration snapshots in the datastore.
+# Do not modify the definitions of CMD keys that already exist.
+CMD_CREATE_NEW = 'create_new'
```

Comment:
Must these be in feconf? Can they live only in rights_domain?

If a refactor is required subsequently (due to imports) then maybe add a TODO/issue to get rid of these when that happens.

Reply:
Done.